custom_components/adaptive_lighting/switch.py

```diff
@@ -589,7 +589,7 @@ def validate(
     if defaults is None:
         data = {key: default for key, default, _ in VALIDATION_TUPLES}
     else:
-        data = defaults
+        data = deepcopy(defaults)
```

Comment:
This was the issue @th3w1zard1, we were mutating the defaults `switch._config_backup` dict.

Reply:
Oops! I'm glad you caught this!